src/Itinerary.js

```diff
@@ -0,0 +1,12 @@
+const { blue } = require("color-name");
```

Comment:
Why is this here? You arent using it anywhere in the file so you can delete this